### PR TITLE
Site Template Basic redesign

### PR DESCRIPTION
**Pull request by bartoszglow**
Redesign for Site Template Basic.

Introduces new:
- wireframe.xd
- previews
- theme
- content package

These are breaking changes so it requires to release new major version of  Site Template Basic (v5.0.0).

### CODE REVIEW

**Review by bpauli — COMMENTED** [review]
General observations during review:
- search is not working
- https://github.com/adobe/aem-site-template-basic/blob/main/site/src/main/content/jcr_root/content/aem-site-template-basic/.content.xml#L13 should point to the new home.html
- The home on the same level as the other navigation items looks weird to me. I would rather keep the old structure and remove the home navigation item. It's more common to link to the home via the logo. (needs to be discussed with @gabrielwalt)

**Comment by gabrielwalt** [comment]
The theme isn't yet responsive. I think this should be added in a next iteration and not hold back merging the current work.

**Review by vladbailescu — APPROVED** [review]

File: theme/src/site/_variables.scss [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
 
-$color-foreground:      #202020;
-$color-background:      #FFFFFF;
-$color-link:            #2020E0;
+$color-foreground: #202020;
+$color-background: #ffffff;
+$color-link: #2020e0;
+$accent: #2954ff;
+$accent-focus: #436eff;
+$accent-hover: #103be6;
```
Comment (godanny86) [inline theme/src/site/_variables.scss:16]:
```suggestion
$accent: #2954ff;
$accent-focus: lighten($accent, 10%);
$accent-hover: darken($accent, 10%);
```

File: theme/src/site/_variables.scss [hunk 2]
```diff
@@ -1,15 +1,32 @@
...
+$grey-bar: #eaebeb;
+$grey1: #eceff1;
+$grey2: #cfd8dc;
+$grey3: #607d8b;
+$grey4: #455a65;
+$color-shadow: #00000029;
+$error: #d7373f;
+$error-hover: #be1e26;
+$success: #268e6c;
+$success-hover: #0d7553;
```
Comment (godanny86) [inline theme/src/site/_variables.scss:28]:
```suggestion
$error: #d7373f;
$error-hover: darken($error, 10%);
$success: #268e6c;
$success-hover: darken($success, 10%);
```

Could use some Sass power to limit the number of color variables needed...

File: theme/src/components/text/_text.scss [hunk 3]
```diff
@@ -1,2 +1,43 @@
...
+    }
+    &:focus {
+      text-decoration: underline;
+      border-color: $accent;
+      outline: none;
+      text-decoration-thickness: 0.125em;
+      text-decoration-color: $accent;
+      text-decoration-style: solid;
+    }
+  }
```
Comment (godanny86) [inline theme/src/components/text/_text.scss:29]:
```suggestion
  & p a {
    color: $color-link;
    text-decoration: none;
    border: 0.125em solid $transparent;
    border-radius: 0.25em;
    &:hover {
      color: darken($color-link, 20%);
      text-decoration: underline;
      text-decoration-thickness: 0.125em;
      text-decoration-color: darken($color-link, 20%);
      text-decoration-style: solid;
    }
    &:focus {
      text-decoration: underline;
      border-color: $accent;
      outline: none;
      text-decoration-thickness: 0.125em;
      text-decoration-color: $color-link;
      text-decoration-style: solid;
    }
  }
```

I think the `$color-link` variable is more appropriate? `$accent` seems like the Brand color, while `$color-link` is the color of anchor tags...

**Comment by godanny86** [comment]
Content Page Template does not have the client libraries applied: `/conf/wknd/settings/wcm/templates/content-page/structure.html`

image

**Comment by godanny86** [comment]
It seems like the Style system for Title policy is inconsistent. Why is there a `h1 style`? Shouldn't that just be the default for the `h1` element? `Content Title` and `Page Title` seem like styles to me but here they are separate policies...

image

**Comment by bartoszglow** [comment]
thanks @godanny86 for your review. I will merge and release new version of the site template however will prepare an issue for things you have listed.

**Pull request merged**